app/templates/docs/scim.html

```diff
@@ -0,0 +1,422 @@
+      "value": "58d54ee29a0af8d02877959e",
+      "type": "Board",
+      "displayName": "Sweet"
+    }
+  ]
+}
+</code></pre></div>
+
+<h2 id="routes">Routes</h2>
+<h3>List and search</h3>
+
+<table>
+  <tr>
+    <th>Users</th>
+    <th>GET /scim/v2/Users</th>
+  </tr>
+  <tr>
+    <td>Groups</td>
+    <td>GET /scim/v2/Groups</td>
+  </tr>
+  <tr>
+    <td>All resources including Users and Groups</td>
+    <td>GET /scim/v2</td>
+    <td>You can use meta.resourceType ("user", "group", etc) or meta.trelloType ("member", "team", "board", etc) to filter by types.</td>
+  </tr>
+</table>
+
+<h4>Parameters</h4>
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Type</th> 
+    <th>Description</th>
+    <th>Example</th>
+  </tr>
+  <tr>
+    <td>filter</td>
+    <td>string</td>
+    <td>Filter the resources using an expressive but simple query language. See <a href="#filtering">Filtering section</a>, below.</td>
+    <td>username eq "bob"</td>
+  </tr>
+  <tr>
+    <td>count</td>
+    <td>integer</td>
+    <td>The maximum number of results to include. Used with startIndex for pagination.</td>
+    <td>20</td>
+  </tr>
+  <tr>
+    <td>startIndex</td>
+    <td>integer</td>
+    <td>The number of the first result to include. Used with count for pagination.</td>
+    <td>30</td>
+  </tr>
+  <tr>
+    <td>sortBy</td>
+    <td>string</td>
+    <td>An attribute name to sort the results by.</td>
+    <td>groups.value</td>
+  </tr>
+  <tr>
+    <td>sortOrder</td>
+    <td>string</td>
+    <td>Either ascending (default) or descending.</td>
+    <td>descending</td>
+  </tr>
+</table>
+<h3>Get a single User, Team or Board</h3>
+<h4>User</h4>
+<code>GET /scim/v2/Users/USER_ID</code>
+e.g. <code>/scim/v2/Users/58d11883e7ea028acc5b19a4</code>
+<h4>Group (Teams and Boards)</h4>
+<code>GET /scim/v2/Groups/GROUP_ID</code>
+
+<h3>Create a new User</h3>
+<code>POST /scim/v2/Users</code>
+<p>sending a header of <code>Content-type: application/json</code>
+and a body with a User resource representation, such as:</p>
+<code>{"displayName": "David Walliams", "emails": [ { "value": "dw@example.com" } ] }</code>
+<p>The following attributes will be used; any other will be ignored:</p>
+<ul>
+  <li>Required:
+    <ul>
+      <li>one of displayName, name.formatted, or name.givenName and name.familyName</li>
+      <li>emails.value</li>
+    </ul>
+  </li>
+  <li>Optional:
+    <ul>
+      <li>userName</li>
+      <li>Normally best to omit, and one will be autogenerated. If supplied, must be 3-100 characters of only lowercase letters, underscores and numbers.</li>
+      <li>locale</li>
+      <li>timezone</li>
+    </ul>
+  </li>
+</ul>
+
+<p>If successful, the response will have a status of 201, and contain the full User resource in the body.</p>
+<p>The created Trello user will be owned by the enterprise. If the enterprise has domains configured, the user will not email address require confirmation. Otherwise, an email will be sent asking the user to confirm their address.</p>
```

Comment:
Need to add something like
> The user will be enabled for Single Sign On if the enterprise has this configured. Otherwise, the user will need to [reset their password](https://trello.com/forgot) to set a password for login.